utilities/pipelines/dependencies/Microsoft.KeyVault/vaults/parameters/nopr.parameters.json

```diff
@@ -0,0 +1,63 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "name": {
+      "value": "adp-<<namePrefix>>-az-kv-nopr-001"  // keyvault with purge protection enabled, to be used for encryption
+    },
+    "accessPolicies": {
+      "value": [
+        {
+          "tenantId": "<<tenantId>>",
+          "objectId": "bab643ad-2988-4560-9f2b-c8282ba345f0", // Backup Management Service needs access to back up the keys and secrets, along with the associated VMs
```

Comment:
are the access policies the minimum ones? is the backup management needed?

Reply:
Backp management is not needed so removed